### PR TITLE
GOLDEN BIKE HORN CAN NOW HONK TWICE AS FAST AS A REGULAR BIKE HORN

### PR DESCRIPTION
**Pull request by Tenryumi**
# HONK

image
# WHAT WAS HONKED:
- HONK

_Purpose: HONK_
#### HONKLOG

:cl: Honk
rscadd: The golden bike horn can now honk twice as rapidly as a regular bike horn. God help us all. HONK!
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/clown_items.dm [hunk 1]
```diff
@@ -148,6 +148,7 @@
 	desc = "Golden? Clearly, its made with bananium! Honk!"
 	icon_state = "gold_horn"
 	item_state = "gold_horn"
+	cooldowntime = 10
```
Comment (Time-Green) [inline code/game/objects/items/weapons/clown_items.dm]: Excuse me, but wouldn't this make it quintuple as fast as the original?

**Comment by ShadowDeath6** [comment]
powercreep

**Comment by Cruix** [comment]
Absolutely unacceptable. This will completely fuck blobs even more than they already are, screw over revheads early and late game, contribute to the growing holoparasite problem, reinforce the emag-revolver meta, and destroy the viability of shadowlings. I don't know how you could even consider this change.

Did it ever occur to you how much this would affect traitors, abductors, changelings, nukeops, deathsquads, and megafauna? This game is a complex balancing act of interwoven checks and balances, maintained by some of the most highly trained and educated spaceman experts in the world, and this PR shows complete disregard for that haphazard yet workable tapestry which we have spent five years holding together with duct tape and bubblegum.

If by some miracle this is merged we may as well set the probability of all gamemodes to zero except extended so that the playerbase can at least eke out some meager entertainment from roleplay, since any semblance of balance will be utterly and completely destroyed.

All in all decent change :+1:

**Comment by ShadowDeath6** [comment]
New copypasta

**Pull request merged**